src/app/(authenticated)/[username]/page.tsx

```diff
@@ -12,8 +16,46 @@ export default async function page({ params }: PageProps) {
   return (
     <Container>
       <div className="flex">
-        <Avatar avatarUrl={userResult.value.avatar}></Avatar>
-        <h1> {userResult.value.displayName}</h1>
+        <div className="w-1/4">
```

Comment:
- Generally, when you have a small component like the avatar, it's better to give it a fixed width (like `w-20`) instead of a relative width (like `w-1/4`).
The reason is that a fixed width is more predictable.